src/Core/JVM/ClassInvokerInterface.php

```diff
@@ -10,15 +14,18 @@ public function __construct(JavaClassInterface $javaClass, array $options);
 
     public function getJavaClass(): JavaClassInterface;
 
+    /**
+     * @param mixed ...$arguments
+     */
     public function construct(...$arguments): ClassInvokerInterface;
```

Comment:
This redundant type annotation doesn't help anything.

Reply:
Sorry! it is auto-generated :sob: